### DISCUSSION TITLE
OpenAI - OpenAI Spec file problems

### DISCUSSION THREAD

**Discussion by Mistobaan** [ogen-go/ogen#1166] [Q&A]
I am trying to generate the client code for *some* of these endpoints:

https://raw.githubusercontent.com/openai/openai-openapi/master/openapi.yaml

but I am getting these errors:
- discriminator inference not implemented
- operation has no \"default\" response

Any insights on what are those errors ? these are the configuration I am using

generate.go
```
//go:generate go run github.com/ogen-go/ogen/cmd/ogen@latest --target client --clean openapi.gen.yaml
```

ogen.yml
```
# sets parser options.
parser:
  # enables type inference for schemas. Schema parser will try to detect schema type by its properties.
  infer_types: false
  #  enables remote references resolving. See https://github.com/ogen-go/ogen/issues/385.
  allow_remote: true
  # is maximum depth of schema generation. Default is 1000.
  depth_limit: 1000

# sets generator options.
generator:
  features:
    enable:
      - "debug/example_tests"
    disable:
      - 'paths/server'
  # functions to ignore
  ignore_not_implemented: ['all']

# is a path to expanded spec.
expand: ""
```

**Comment by ernado** [comment]
We have openai api in examples:

https://github.com/ogen-go/ogen/blob/main/_testdata/examples/openai.yml

But patch was required to make it work: https://github.com/ogen-go/ogen/commit/d7af5d8c7d16ccdfa457b0eecdc8a14ca6c9ece8

Hope this will get you some hints. If not, let us know and we wil try to help.
